BUILD_INSTRUCTIONS.md

```diff
@@ -40,7 +51,7 @@ Libraries could be installed through yum.
         libyaml-devel       0.1.1
         flex-devel          2.5.35
 ## Build
-After installed the libraries listed above, get code with command:
+After installed the libraries listed above, get source code:
```

Comment:
"After installed the libraries listed above, get source code" => "After the libraries list above are installed, get source code"